.github/workflows/web.yml

```diff
@@ -1,4 +1,6 @@
 name: web
+run-name: ${{ github.event_name == 'schedule' || (github.event_name == 'dispatch' && inputs.is_nightly == 'true') && format('web nightly {0}', github.ref_name) }}
```

Comment:
I'm not sure, but maybe dispatch should be replaced by workflow_dispatch

Reply:
![image](https://github.com/centreon/centreon/assets/144020537/135c77ba-d387-43ed-abe9-2daa39a1d914)
with the actual update we have this output , but while adding && inputs.is_nightly == 'true' we will get false is it necessary to adding it